tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -85,7 +85,7 @@ public void testDescribeWithKRaftAndBootstrapControllers(ClusterInstance cluster
 
         // Change expected message to reflect latest MetadataVersion (SupportedMaxVersion increases when adding a new version)
         assertEquals("Feature: metadata.version\tSupportedMinVersion: 3.0-IV1\t" +
-                "SupportedMaxVersion: 4.0-IV0\tFinalizedVersionLevel: 3.7-IV4\t", outputWithoutEpoch(features.get(1)));
+                "SupportedMaxVersion: 4.0-IV0\tFinalizedVersionLevel: 3.8-IV0\t", outputWithoutEpoch(features.get(1)));
```

Comment:
This needed to change as well as we changed the parameter of the test to 3.8-IV0